plugins/woocommerce-blocks/assets/js/blocks/product-elements/add-to-cart-form/edit.tsx

```diff
@@ -59,8 +60,23 @@ const Edit = ( props: BlockEditProps< Attributes > ) => {
 		[]
 	);
 
+	function handleNoticeRemove() {
+		setAttributes( { isNoticeHidden: true } );
+	}
+
 	return (
 		<>
+			<InspectorControls>
+				{ ! attributes.isNoticeHidden && (
+					<InspectorNotice onRemove={ handleNoticeRemove }>
+						{ __(
+							"This block's appearance and behavior may differ depending on a product, e.g. type, quantity, variation.",
```

Comment:
I see the design was like that, but shouldn't this be "on the product"? Otherwise it feels a bit weird to me as it's not clear it refers to the product being displayed in the Single Product template. :thinking: 

```suggestion
							"This block's appearance and behavior may differ depending on the product, e.g. type, quantity, variation.",
```